static/js/redux/ui/modals/import_sis_modal.jsx

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright (C) 2017 Semester.ly Technologies, LLC
+
+Semester.ly is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Semester.ly is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+*/
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Modal from 'boron/WaveModal';
+import * as SemesterlyPropTypes from '../../constants/semesterlyPropTypes';
+
+class ImportSISModal extends React.Component {
+  componentDidMount() {
+    if (this.props.isVisible) {
+      this.modal.show();
+    }
+  }
+
+  componentDidUpdate() {
+    if (this.props.isVisible) {
+      this.modal.show();
+    }
+  }
+
+  importSIS() {
+      const form = document.createElement('form');
+      form.method = 'post';
+      form.action = 'http://sisdevelopment.sis.jhu.edu/semtest/';
+      form.encType = 'application/x-www-form-urlencoded';
+      document.body.appendChild(form);
+      const input = document.createElement('input');
+      input.name = 'data';
```

Comment:
lets make the button a form and use ajax. ref: https://stackoverflow.com/questions/42352941/how-to-send-input-hidden-in-react-js